blockchain/blockchain-wallet-v4-frontend
# Feat/interest txn history

## Description (optional)
* Add Interest transaction history download button to convert selected coin's interest transactions into a csv file.
* **Not Asked:**
<img width="1595" alt="Screen Shot 2021-04-23 at 12 18 19 AM" src="https://user-images.githubusercontent.com/53186076/115817915-b972e200-a3c9-11eb-9a6d-d799b9499ea4.png">

* **Loading:**
<img width="1614" alt="Screen Shot 2021-04-23 at 12 24 17 AM" src="https://user-images.githubusercontent.com/53186076/115818188-4158ec00-a3ca-11eb-9edb-b7bcd0e639d8.png">

* **Success:**
<img width="1648" alt="Screen Shot 2021-04-23 at 12 18 28 AM" src="https://user-images.githubusercontent.com/53186076/115818101-0951a900-a3ca-11eb-91fd-30cf7b539a10.png">

* **Success after saving csv:**
<img width="1636" alt="Screen Shot 2021-04-23 at 12 18 35 AM" src="https://user-images.githubusercontent.com/53186076/115818112-0d7dc680-a3ca-11eb-8da3-10f6c44ce085.png">

* **Failed:**
<img width="1644" alt="Screen Shot 2021-04-23 at 12 19 16 AM" src="https://user-images.githubusercontent.com/53186076/115818134-1cfd0f80-a3ca-11eb-82f7-5a41e3139f2b.png">

## Testing Steps (optional)
* Login into your wallet account with interest transactions
* Click Earn Interest on the top navigation
* Click Transaction History tab
* After initial transactions load, download button should show
* Click download
* Wait for data to finish fetching
* Click Save Report
* Should see button change to Saved
